packages/plugin-sharp/src/index.ts

```diff
@@ -1,15 +1,11 @@
-import loadImageSharp from "./functions/loadImage";
-import resizeImageSharp from "./functions/resize";
+import * as props from "./functions";
 
-export default function CanvacordMiddleware() {
+export default function SharpPlugin() {
     return (pluginsManager: any) => {
-        pluginsManager.register("@canvacord/plugin-sharp", {
-            props: {
-                resizeImage: resizeImageSharp,
-                loadImage: loadImageSharp
-            }
-        });
+        pluginsManager.register("@canvacord/plugin-sharp", { props });
```

Comment:
imo, using function names like `blur` is always superior to `blurImage` in this library since that's what we have been doing for years.